src/java/picard/analysis/directed/HsMetrics.java

```diff
@@ -107,15 +110,26 @@
     /** The fold by which the baited region has been amplified above genomic background. */
     public double FOLD_ENRICHMENT;
 
-    /** The number of targets that did not reach coverage=2 over any base. */
+    /** The number of targets that did not reach coverage=1 over any base. */
     public double ZERO_CVG_TARGETS_PCT;
 
+    /** The fraction of aligned bases that were filtered out because they were in reads with low mapping quality (default is < 0). */
```

Comment:
I think that the "default is < 0 "comment is confusing. the metric file should have a header with the parameters used, so perhaps you can refer to that instead of stating the default which may be irrelevant. similarly below.